rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -957,6 +957,20 @@ boolean pauseJob(
                     throws NotConnectedRestException, UnknownJobRestException, PermissionRestException;
 
     /**
+     * Restart all tasks in error in the job represented by jobid
+     * @param sessionId a valid session id
+     * @param jobId the id of the job
+     * @return true if success, false if not
+     */
+    @PUT
+    @Path("jobs/{jobid}/restartAllInErrorTasks")
+    @Produces("application/json")
+    boolean restartAllInErrorTasks(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.ow2.proactive_grid_cloud_portal.scheduler.exception.NotConnectedRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.UnknownJobRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.PermissionRestException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)